resources/finalize_recording.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+RECORDINGS_DIR=$1
+
+echo "This is a dummy finalize script" > /tmp/finalize.out
+echo "The script was invoked with recordings directory $RECORDINGS_DIR." >> /tmp/finalize.out
+echo "You should put any finalize logic (renaming, uploading to a service" >> /tmp/finalize.out
+echo "or storage provider, etc.) in this script" >> /tmp/finalize.out
+
+exit 0
```

Comment:
Why does this dummy script exist?

Reply:
so that the code can assume the file will exist and it's clearer what a user needs to do to write their own